ads/google/a4a/docs/RTB-Implementation.md

```diff
@@ -0,0 +1,65 @@
+ 
+**Site Object Additional field: amp**
+A new field is added to the 'Site' object of the OpenRTB standard to indicate whether a webpage is built on AMP HTML.  In OpenRTB 2.5, this is section 3.2.13.
+ 
+![RTB1](./img1.png)
+ 
+**Imp Object Additional field: ampad**
+A new field is added to the 'Imp' object of the OpenRTB standard to provide more detail around AMP ad requirements and how AMP ads will load.  In OpenRTB 2.5, this is section 3.2.4.
+ 
+![RTB2](./img2.png)
+ 
+**AMP Ad Status Table**
+ 
+![RTB3](./img3.png)
+ 
+### RTB Bid Response
+ 
+SSPs will need to provide a new field in the bid response to allow bidders to return AMP Ad content, and RTB bidders will need to populate that field in order to return AMP Ads.  As of [OpenRTB 2.5](http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943812000&usg=AFQjCNEQipmPbKkfCuGkKvzgdpC-I60EOg), this is not yet included in the spec, but the proposed workflow is a new field that accepts a URL pointing to AMP ad content.  
+ 
+**AMP Ad Status Table**
+ 
+![RTB4](./img4.png)
+ 
+### Verification of valid AMP
+ 
+* For AMP Ads to be early rendered, the exchange is required to verify and sign in real time that the ad is written in amp4ads  <html amp4ads> creative format.
+* See "[Proposed Design](https://github.com/ampproject/amphtml/issues/3133&sa=D&ust=1496436943832000&usg=AFQjCNGwYUpiQAwBoxZLaBDfmDYurYeTWw)" for signing.
+* Ads that are valid AMP Ads will be allowed to render early by AMP pages.  Ads that are not verified as valid AMP Ads  will render at the same speed as non-AMP ads.
+* Only AMP Ads  should be returned in the ampadurl .
+ 
+### Server-side fetch
+ 
+* For AMP Ads to be early rendered, AMP ad content must be fetched with 0 additional "hops" from the client.  This is designed to avoid poor user experiences due to ad latency and extra client-side calls.
+* The exchange's servers (not the client browser) will request the AMP ad content located at the URL provided in ampadurl  after a bidder wins the auction.
+* Creative servers must respond and return content within some reasonable SLA, recommended at 150ms.
+* The AMP Ad will be injected into the adslot and subsequently rendered.  Note that since a valid AMP Ad cannot contain an iframe or another ad tag, the server-side fetch must retrieve the actual HTML of the creative.
+ 
+### Impression Tracking and Billing URLs
+ 
+* RTB buyers often include impression trackers as a structured field in the bid response; e.g. Bid.burl  -- the "billing notice URL" -- in OpenRTB 2.5.
```

Comment:
Rewrite as:

... response (for example, `Bid.burl`, the "billing notice URL" in OpenRTB 2.5).